scripts/storybook/webpack.config.js

```diff
@@ -86,5 +89,33 @@ module.exports = (/** @type {webpack.Configuration} */ config) => {
 
   config.optimization.minimize = false;
 
+  if (process.env.TF_BUILD) {
+    console.log(
+      'Storybook webpack config:',
+      // Plugins have circular references, and they're probably not the issue here
+      JSON.stringify(config, (key, value) => (key === 'plugins' ? undefined : value), 2),
+    );
+    const northstarPath = path.join(gitRoot, 'packages/fluentui/react-northstar');
+    const filesToWatch = [
+      path.join(gitRoot, 'tsconfig.json'),
+      path.join(northstarPath, 'tsconfig.json'),
+      path.join(northstarPath, 'package.json'),
+    ];
+    for (const file of filesToWatch) {
+      fs.watchFile(file, (curr, prev) => {
+        let stack;
+        try {
```

Comment:
you don't need to try/catch, just creating a new error is enough to get the stack trace